### ISSUE TITLE
Add an Observing System Monitor webpage to robonet_site

### ISSUE THREAD

**Issue by rachel3834** [ytsapras/robonet_site#25]
Page should include req_vs_obs plots for ROME and REA over the -24hrs->+24hrs from now. 
Plot of Moon proximity to mid-Bulge with point size scaled by illumination fraction.

Store information on completed observations

**Comment by rachel3834** [comment]
I've added code to regularly poll Valhalla for the status of all subrequests for all our Obsrequests, ingest this information into the DB.  This code is fully API-based.

**Comment by rachel3834** [comment]
Update to code: initial version compiled a list of all existing active obs requests, then queried Valhalla separately for the subrequests associated with each request.  Since this software runs every 2hrs under the cron, this resulted in a large number of Valhalla queries per day, and I found we were hitting the maximum allowed number.  Valhalla has a limit of 5000 hits per user per day to avoid robots impacting its performance.  Unfortunately, once this limit is hit, the user can no longer submit obs requests either.  

I've now restructured the code so that it queries only once for all observation requests, each time its run, and I will restrict the number of times that it runs under the cron.
